modulefiles/tasks/cheyenne/python_srw.lua

```diff
@@ -3,3 +3,10 @@ prepend_path("MODULEPATH","/glade/work/epicufsrt/contrib/miniconda3/modulefiles"
 load(pathJoin("miniconda3", os.getenv("miniconda3_ver") or "4.12.0"))
 
 setenv("SRW_ENV", "regional_workflow")
+
+local mod_path, mod_file = splitFileName(myFileName())
+local uwtools_scripts_path =  pathJoin(mod_path, "../../../ush/python_utils/uwtools")
```

Comment:
I'm curious if there is a reason why the scripts directory isn't at the end of this path? Doing that will cleanup the imports in the modified scripts under the `ush` directory to be `from templater import set_template` instead of `from scripts.templater import set_template`. It looks like PEP8 is okay with the latter but that's with subclasses and not a directory.

Reply:
I'm fine with the suggested changes of leaving it as it is and adding a comment. I wasn't aware that other things like a driver could be imported, so I understand the reasoning. And I do think when a full integration of the tools occurs a cleaner solution will appear.